### ISSUE TITLE
MPD not starting after 6.95.3 update

### ISSUE THREAD

**Issue by Hollol** [OpenELEC/OpenELEC.tv#4944]
After updating to Openelec 6.95.3 my mpd setup stopped working. Actually the mpd is not starting any more. 
Calling mpd manually from shell gives me the following error:

`mpd: error while loading shared libraries: libogg.so.0: cannot open shared object file: No such file or directory`

So it seems, a shared library is missing after the update.

**Comment by Hollol** [comment]
I do not know, if it is welcome here to post this, but the above error is actually also appearing in LE with already a solution:

https://github.com/LibreELEC/LibreELEC.tv/pull/334

**Comment by Hollol** [comment]
I do not know, if it is welcome here to post this, but the above error is actually also appearing in LE with already a solution:

LibreELEC/LibreELEC.tv#334

**Comment by ManuRhombea** [comment]
I've found another feature failed due to this file missing: extracting audio CDs in Flac.

1. Install the add-on "Flac Audio Encoder" from Kodi official repository.
2. Configure Kodi to encode audio discs in Flac after insertion.
3. Wait and look the nice error pop-up "Flac audio encoder can't load Dll" on this Linux system ;-)

Reading the logfile in .kodi/temp/kodi.log (in zip attached, line 717), I see: `ERROR: Unable to load /storage/.kodi/addons/audioencoder.flac/audioencoder.flac.so, reason: libogg.so.0: cannot open shared object file: No such file or directory` 
I've not found this library in /lib or /usr/lib.

[kodi.zip](https://github.com/OpenELEC/OpenELEC.tv/files/664187/kodi.zip)

**Comment by EpicVoyage** [comment]
There is a similar issue in 7.0.1:

    # mpd
    mpd: error while loading shared libraries: libFLAC.so.8: cannot open shared object file: No such file or directory

**Comment by CvH** [comment]
you have to rebuild them that they work with the current release, basically nobody cares about the unofficial addons at the "moment" so don't expect them working

**Comment by EpicVoyage** [comment]
Yeah, I expected that. This issue is "open" and I found it while checking who else had run into the same issue.

I have been able to work around it for now by creating the directory `/storage/.kodi/addons/service.multimedia.mpd/lib` and copying these 64-bit libraries to it from my machine: libFLAC.so.8, libfaad.so.2, libmad.so.0, libogg.so.0
